chromium_src/components/crash/content/app/crashpad_win.cc

```diff
@@ -55,9 +55,16 @@ bool BraveCrashpadClient::StartHandler(
     const std::vector<std::string>& arguments,
     bool restartable,
     bool asynchronous_start) {
+
```

Comment:
you don't need to do anything here, it's already handled by `env->GetVar(kServerUrlVar, &url);`

Reply:
At the very least we need to change our hardcoded url to use the `url` variable they pass in... Also, isn't a bit confusing to have a separate environment variable just for Windows?

Reply:
If we do this, we also need to account for when they pass in `"https://clients2.google.com/cr/report` as `url`. The amount of logic will be the same, and more confusing.